[2023-06-19T22:02:02.006168+02:00] php.DEBUG: Warning: filemtime(): stat failed for D:\Symfony\NatureWeb\NatureWeb\config\routes\framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"D:\\Symfony\\NatureWeb\\vendor\\symfony\\config\\Resource\\FileResource.php","line":61,"trace":[{"file":"D:\\Symfony\\NatureWeb\\vendor\\symfony\\config\\Resource\\SelfCheckingResourceChecker.php","line":44,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2023-06-19T22:02:02.042640+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at D:\\Symfony\\NatureWeb\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2023-06-19T22:02:02.209982+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f19043"},"request_uri":"http://localhost:8000/_wdt/f19043","method":"GET"} []
[2023-06-19T22:04:29.943252+02:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"f19043"},"request_uri":"http://localhost:8000/_profiler/f19043?panel=security","method":"GET"} []
[2023-06-28T19:23:50.244354+02:00] console.CRITICAL: Error thrown while running command ""server:un"". Message: "Command "server:un" is not defined.  Did you mean one of these?     dbal:run-sql     make:functional-test     make:unit-test     server:dump     server:log" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"server:un\" is not defined.\n\nDid you mean one of these?\n    dbal:run-sql\n    make:functional-test\n    make:unit-test\n    server:dump\n    server:log at D:\\Symfony\\NatureWeb\\vendor\\symfony\\console\\Application.php:694)","command":"\"server:un\"","message":"Command \"server:un\" is not defined.\n\nDid you mean one of these?\n    dbal:run-sql\n    make:functional-test\n    make:unit-test\n    server:dump\n    server:log"} []
[2023-06-28T19:23:58.061416+02:00] console.CRITICAL: Error thrown while running command ""server:run"". Message: "Command "server:run" is not defined.  Did you mean one of these?     dbal:run-sql     server:dump     server:log" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    dbal:run-sql\n    server:dump\n    server:log at D:\\Symfony\\NatureWeb\\vendor\\symfony\\console\\Application.php:694)","command":"\"server:run\"","message":"Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    dbal:run-sql\n    server:dump\n    server:log"} []
[2023-06-28T19:24:21.548163+02:00] console.CRITICAL: Error thrown while running command ""server:run"". Message: "Command "server:run" is not defined.  Did you mean one of these?     dbal:run-sql     server:dump     server:log" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    dbal:run-sql\n    server:dump\n    server:log at D:\\Symfony\\NatureWeb\\vendor\\symfony\\console\\Application.php:694)","command":"\"server:run\"","message":"Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    dbal:run-sql\n    server:dump\n    server:log"} []
[2023-06-28T19:30:16.712929+02:00] console.CRITICAL: Error thrown while running command ""server:run"". Message: "Command "server:run" is not defined.  Did you mean one of these?     dbal:run-sql     server:dump     server:log" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    dbal:run-sql\n    server:dump\n    server:log at D:\\Symfony\\NatureWeb\\vendor\\symfony\\console\\Application.php:694)","command":"\"server:run\"","message":"Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    dbal:run-sql\n    server:dump\n    server:log"} []
[2023-06-28T19:32:38.901882+02:00] console.CRITICAL: Error thrown while running command ""server:run"". Message: "Command "server:run" is not defined.  Did you mean one of these?     dbal:run-sql     server:dump     server:log" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    dbal:run-sql\n    server:dump\n    server:log at D:\\Symfony\\NatureWeb\\vendor\\symfony\\console\\Application.php:694)","command":"\"server:run\"","message":"Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    dbal:run-sql\n    server:dump\n    server:log"} []
[2023-06-28T19:39:02.490960+02:00] console.CRITICAL: Error thrown while running command ""server:run"". Message: "Command "server:run" is not defined.  Did you mean one of these?     dbal:run-sql     server:dump     server:log" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    dbal:run-sql\n    server:dump\n    server:log at D:\\Symfony\\NatureWeb\\vendor\\symfony\\console\\Application.php:694)","command":"\"server:run\"","message":"Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    dbal:run-sql\n    server:dump\n    server:log"} []
[2023-06-28T19:39:18.676705+02:00] console.CRITICAL: Error thrown while running command ""server:start"". Message: "Command "server:start" is not defined.  Did you mean one of these?     server:dump     server:log" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"server:start\" is not defined.\n\nDid you mean one of these?\n    server:dump\n    server:log at D:\\Symfony\\NatureWeb\\vendor\\symfony\\console\\Application.php:694)","command":"\"server:start\"","message":"Command \"server:start\" is not defined.\n\nDid you mean one of these?\n    server:dump\n    server:log"} []
[2023-06-28T19:49:47.109724+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at D:\\Symfony\\NatureWeb\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2023-06-28T19:49:47.275459+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"b658ad"},"request_uri":"http://localhost:8000/_wdt/b658ad","method":"GET"} []
[2023-06-28T20:21:02.951373+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at D:\\Symfony\\NatureWeb\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2023-06-28T20:21:03.145126+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"cb17b6"},"request_uri":"http://localhost:8000/_wdt/cb17b6","method":"GET"} []
[2023-06-28T20:21:34.647564+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at D:\\Symfony\\NatureWeb\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2023-06-28T20:21:34.793380+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"ecbc73"},"request_uri":"http://localhost:8000/_wdt/ecbc73","method":"GET"} []
[2023-06-28T20:21:36.876968+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at D:\\Symfony\\NatureWeb\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2023-06-28T20:21:37.016867+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"451a86"},"request_uri":"http://localhost:8000/_wdt/451a86","method":"GET"} []
[2023-06-28T20:21:37.801788+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at D:\\Symfony\\NatureWeb\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2023-06-28T20:21:37.938264+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"bcb8ce"},"request_uri":"http://localhost:8000/_wdt/bcb8ce","method":"GET"} []
[2023-06-28T20:21:38.706827+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at D:\\Symfony\\NatureWeb\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2023-06-28T20:21:38.849240+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a18934"},"request_uri":"http://localhost:8000/_wdt/a18934","method":"GET"} []
[2023-06-28T20:21:39.870240+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at D:\\Symfony\\NatureWeb\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2023-06-28T20:21:40.004553+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"819f47"},"request_uri":"http://localhost:8000/_wdt/819f47","method":"GET"} []
[2023-06-28T20:21:57.327489+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at D:\\Symfony\\NatureWeb\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2023-06-28T20:21:57.519279+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"162011"},"request_uri":"http://localhost:8000/_wdt/162011","method":"GET"} []
[2023-06-28T20:22:40.875066+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"app\\controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-28T20:22:40.881179+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-28T20:30:35.087831+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"app\\controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-28T20:30:35.090336+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-28T20:30:35.108261+02:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Controller "app\controller\HomeController" has required constructor arguments and does not exist in the container. Did you forget to define the controller as a service?" at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 81 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Controller \"app\\controller\\HomeController\" has required constructor arguments and does not exist in the container. Did you forget to define the controller as a service? at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:81)\n[previous exception] [object] (InvalidArgumentException(code: 0): Controller \"app\\controller\\HomeController\" has required constructor arguments and does not exist in the container. Did you forget to define the controller as a service? at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\Controller\\ContainerControllerResolver.php:54)\n[previous exception] [object] (ArgumentCountError(code: 0): Too few arguments to function App\\Controller\\HomeController::__construct(), 0 passed in D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php on line 136 and exactly 1 expected at D:\\Symfony\\NatureWeb\\src\\Controller\\HomeController.php:8)"} []
[2023-06-28T20:30:35.447047+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"2463c2"},"request_uri":"http://localhost:8000/_wdt/2463c2","method":"GET"} []
[2023-06-28T20:49:55.046754+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"app\\controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-28T20:49:55.048714+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-28T20:49:55.065841+02:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Controller "app\controller\HomeController" has required constructor arguments and does not exist in the container. Did you forget to define the controller as a service?" at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 81 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Controller \"app\\controller\\HomeController\" has required constructor arguments and does not exist in the container. Did you forget to define the controller as a service? at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:81)\n[previous exception] [object] (InvalidArgumentException(code: 0): Controller \"app\\controller\\HomeController\" has required constructor arguments and does not exist in the container. Did you forget to define the controller as a service? at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\Controller\\ContainerControllerResolver.php:54)\n[previous exception] [object] (ArgumentCountError(code: 0): Too few arguments to function App\\Controller\\HomeController::__construct(), 0 passed in D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php on line 136 and exactly 1 expected at D:\\Symfony\\NatureWeb\\src\\Controller\\HomeController.php:8)"} []
[2023-06-28T20:49:55.294709+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"e73a77"},"request_uri":"http://localhost:8000/_wdt/e73a77","method":"GET"} []
[2023-06-28T20:50:24.997134+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"app\\controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-28T20:50:25.002568+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-28T20:50:25.026128+02:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Controller "app\controller\HomeController" does neither exist as service nor as class." at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 81 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Controller \"app\\controller\\HomeController\" does neither exist as service nor as class. at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:81)\n[previous exception] [object] (InvalidArgumentException(code: 0): Controller \"app\\controller\\HomeController\" does neither exist as service nor as class. at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\Controller\\ContainerControllerResolver.php:57)\n[previous exception] [object] (Error(code: 0): Class \"app\\controller\\HomeController\" not found at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:136)"} []
[2023-06-28T20:50:25.231988+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f6ce3c"},"request_uri":"http://localhost:8000/_wdt/f6ce3c","method":"GET"} []
[2023-06-28T20:51:18.447016+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"app\\controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-28T20:51:18.448930+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-28T20:51:18.466142+02:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Controller "app\controller\HomeController" has required constructor arguments and does not exist in the container. Did you forget to define the controller as a service?" at D:\Symfony\NatureWeb\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 81 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Controller \"app\\controller\\HomeController\" has required constructor arguments and does not exist in the container. Did you forget to define the controller as a service? at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:81)\n[previous exception] [object] (InvalidArgumentException(code: 0): Controller \"app\\controller\\HomeController\" has required constructor arguments and does not exist in the container. Did you forget to define the controller as a service? at D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\Controller\\ContainerControllerResolver.php:54)\n[previous exception] [object] (ArgumentCountError(code: 0): Too few arguments to function App\\Controller\\HomeController::__construct(), 0 passed in D:\\Symfony\\NatureWeb\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php on line 136 and exactly 1 expected at D:\\Symfony\\NatureWeb\\src\\Controller\\HomeController.php:13)"} []
[2023-06-28T20:51:18.697086+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"01ca0e"},"request_uri":"http://localhost:8000/_wdt/01ca0e","method":"GET"} []
[2023-06-28T20:51:51.767027+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"app\\controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-28T20:51:51.768946+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-28T20:51:51.786000+02:00] request.CRITICAL: Uncaught PHP Exception ErrorException: "Warning: Undefined property: App\Controller\HomeController::$twig" at D:\Symfony\NatureWeb\src\Controller\HomeController.php line 12 {"exception":"[object] (ErrorException(code: 0): Warning: Undefined property: App\\Controller\\HomeController::$twig at D:\\Symfony\\NatureWeb\\src\\Controller\\HomeController.php:12)"} []
[2023-06-28T20:51:51.969941+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"c3354c"},"request_uri":"http://localhost:8000/_wdt/c3354c","method":"GET"} []
[2023-06-28T20:52:14.415446+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"app\\controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-28T20:52:14.421686+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-28T20:52:46.709293+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"app\\controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-28T20:52:46.715751+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-30T19:46:53.771204+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"app\\controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-30T19:46:53.779070+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-30T19:53:33.038625+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"app\\controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-30T19:53:33.041289+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-30T19:53:33.064258+02:00] request.CRITICAL: Uncaught PHP Exception LogicException: ""app\controller\HomeController" has no container set, did you forget to define it as a service subscriber?" at D:\Symfony\NatureWeb\vendor\symfony\framework-bundle\Controller\ControllerResolver.php line 36 {"exception":"[object] (LogicException(code: 0): \"app\\controller\\HomeController\" has no container set, did you forget to define it as a service subscriber? at D:\\Symfony\\NatureWeb\\vendor\\symfony\\framework-bundle\\Controller\\ControllerResolver.php:36)"} []
[2023-06-30T19:53:33.416774+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"c0f4db"},"request_uri":"http://localhost:8000/_wdt/c0f4db","method":"GET"} []
[2023-06-30T19:54:16.548534+02:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"c0f4db"},"request_uri":"http://localhost:8000/_profiler/c0f4db?panel=logger","method":"GET"} []
[2023-06-30T19:54:22.427763+02:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"c0f4db"},"request_uri":"http://localhost:8000/_profiler/c0f4db?panel=exception","method":"GET"} []
[2023-06-30T19:54:24.053383+02:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"c0f4db"},"request_uri":"http://localhost:8000/_profiler/c0f4db?panel=logger","method":"GET"} []
[2023-06-30T19:56:03.198532+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"app\\controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-30T19:56:03.205723+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-30T19:56:03.242557+02:00] request.CRITICAL: Uncaught PHP Exception LogicException: ""app\controller\HomeController" has no container set, did you forget to define it as a service subscriber?" at D:\Symfony\NatureWeb\vendor\symfony\framework-bundle\Controller\ControllerResolver.php line 36 {"exception":"[object] (LogicException(code: 0): \"app\\controller\\HomeController\" has no container set, did you forget to define it as a service subscriber? at D:\\Symfony\\NatureWeb\\vendor\\symfony\\framework-bundle\\Controller\\ControllerResolver.php:36)"} []
[2023-06-30T19:56:03.510938+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"5118d6"},"request_uri":"http://localhost:8000/_wdt/5118d6","method":"GET"} []
[2023-06-30T20:01:06.541417+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"app\\controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-30T20:01:06.544033+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-30T20:01:06.570504+02:00] request.CRITICAL: Uncaught PHP Exception LogicException: ""app\controller\HomeController" has no container set, did you forget to define it as a service subscriber?" at D:\Symfony\NatureWeb\vendor\symfony\framework-bundle\Controller\ControllerResolver.php line 36 {"exception":"[object] (LogicException(code: 0): \"app\\controller\\HomeController\" has no container set, did you forget to define it as a service subscriber? at D:\\Symfony\\NatureWeb\\vendor\\symfony\\framework-bundle\\Controller\\ControllerResolver.php:36)"} []
[2023-06-30T20:01:06.845314+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"0e1db6"},"request_uri":"http://localhost:8000/_wdt/0e1db6","method":"GET"} []
[2023-06-30T20:02:01.924549+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-30T20:02:01.931701+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-30T20:06:13.238102+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-30T20:06:13.246060+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2023-06-30T20:06:13.377032+02:00] request.CRITICAL: Uncaught PHP Exception Twig\Error\LoaderError: "Unable to find template "home.html.twig" (looked into: D:\Symfony\NatureWeb/templates, D:\Symfony\NatureWeb\vendor\symfony\twig-bridge/Resources/views/Form)." at D:\Symfony\NatureWeb\vendor\twig\twig\src\Loader\FilesystemLoader.php line 227 {"exception":"[object] (Twig\\Error\\LoaderError(code: 0): Unable to find template \"home.html.twig\" (looked into: D:\\Symfony\\NatureWeb/templates, D:\\Symfony\\NatureWeb\\vendor\\symfony\\twig-bridge/Resources/views/Form). at D:\\Symfony\\NatureWeb\\vendor\\twig\\twig\\src\\Loader\\FilesystemLoader.php:227)"} []
[2023-06-30T20:06:13.638573+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"39df08"},"request_uri":"http://localhost:8000/_wdt/39df08","method":"GET"} []
[2023-06-30T20:06:54.354695+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2023-06-30T20:06:54.361313+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
